app/lib/constants/colors.ts

```diff
@@ -89,6 +89,10 @@ export const colors = {
 		conferenceCallEnabledIconBackground: '#156FF5',
 		conferenceCallPhotoBackground: '#E4E7EA',
 		textInputSecondaryBackground: '#E4E7EA',
+		dotBg: '#a9cbff',
+		dotActiveBg: '#1d74f5',
+		gray300: '#5f656e',
```

Comment:
It doesn't seem to be following the new theming system from web.
Maybe they should reevaluate on Figma?

Reply:
There are no such colors in figma, they didn't make that part :(

Reply:
Just to complement, it is part of the calling status that exists on the web, however this status was not layouted in figma.